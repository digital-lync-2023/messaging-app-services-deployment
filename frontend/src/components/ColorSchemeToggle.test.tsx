import { describe, it, expect } from "vitest";
import { ColorSchemeToggle } from "./ColorSchemeToggle";

describe("ColorSchemeToggle", () => {
  it("should render", () => {
    expect(<ColorSchemeToggle />).toMatchSnapshot();
  });
});
