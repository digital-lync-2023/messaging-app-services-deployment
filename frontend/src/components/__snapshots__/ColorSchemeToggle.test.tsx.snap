// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ColorSchemeToggle > should render 1`] = `<ColorSchemeToggle />`;
